standalone/Vagrantfile

```diff
@@ -61,6 +62,30 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
         vb.cpus = options[:cpus] if options[:cpus]
       end
 
+      # provider: openstack
+      node_config.vm.provider :openstack do |os, override|
+        override.ssh.username  = "#{ENV['OS_SSH_USER']}"
+        override.ssh.private_key_path = "#{ENV['OS_SSH_PRIVATE_KEY_PATH']}"
+        os.meta_args_support      = true
+        os.openstack_auth_url     = "#{ENV['OS_AUTH_URL']}"
+        os.username               = "#{ENV['OS_USERNAME']}"
+        os.password               = "#{ENV['OS_PASSWORD']}"
+        os.flavor                 = "#{ENV['OS_FLAVOR']}" # TODO: config
+        os.image                  = "#{ENV['OS_IMAGE']}" # TODO: config
+        os.project_name           = "#{ENV['OS_PROJECT_NAME']}" # TODO: config
```

Comment:
OS_PROJECT_NAME should not be a config/todo either, it comes from the openrc file too.